### PR TITLE
use dlang.org's release build

### PR DESCRIPTION
**Pull request by MartinNowak**

### CODE REVIEW

File: create_dmd_release/create_dmd_release.d [hunk 1]
```diff
@@ -517,8 +517,8 @@ void buildAll(Bits bits, string branch, bool dmdOnly=false)
             if (bits == Bits.bits32)
             {
                 changeDir(cloneDir~"/dlang.org");
-                run(makecmd~" DOC_OUTPUT_DIR="~origDir~"/docs all");
-                copyFile("d.tag", origDir~"/docs/d.tag");
+                run(makecmd~" DOC_OUTPUT_DIR="~origDir~"/docs release");
+                copyFile("d-release.tag", origDir~"/docs/d.tag");
```
Comment (MartinNowak) [inline create_dmd_release/create_dmd_release.d]: Will need to look at reenabling dman soon, for now we're not building it anyhow.

**Comment by wilzbach** [comment]
Yeah we are finally getting rid of these patches :)

**Comment by MartinNowak** [comment]
Will run a full build locally before merging this.

**Comment by MartinNowak** [comment]
Required fixes:
https://github.com/dlang/tools/pull/272
https://github.com/dlang/dlang.org/pull/1957

**Comment by MartinNowak** [comment]
Finally nightlies again, https://nightlies.dlang.org/dmd-master-2017-12-13/.

**Pull request merged**